api/pom.xml

```diff
@@ -36,6 +36,10 @@
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.annotation.versioning</artifactId>
         </dependency>
+        <dependency>
+            <groupId>biz.aQute.bnd</groupId>
+            <artifactId>bnd-baseline-maven-plugin</artifactId>
```

Comment:
IIUC this dependency needed to process the new use of the `@BaselineIgnore` annotation in the source code.

The annotation has retention `CLASS` (not `RUNTIME`) so there is no runtime detection of the annotation.

Perhaps, therefore, this dependency should have scope `provided` to avoid the plug-in bleeding into the runtime class path of anything that depends on the MP Metrics API component.